aliases/aliases.go

```diff
@@ -0,0 +1,127 @@
+package aliases
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	bolt "github.com/coreos/bbolt"
+	"github.com/gomeeseeks/meeseeks-box/db"
+)
+
+var aliasesBucketKey = []byte("aliases")
+
+// ErrAliasNotFound is returned when an alias can't be found
+var ErrAliasNotFound = fmt.Errorf("alias not found")
+
+// Alias represent an alias for a command
+type Alias struct {
+	Alias   string
+	Command string
+	Args    []string
+}
+
+// Add adds a new alias for a user ID
```

Comment:
Rename to Create instead of Add, this way it's symmetric to the rest of the commands.

Reply:
Done.